### PR TITLE
ui: add header component

### PR DESCRIPTION
**Pull request by Danylo-tk**

### CODE REVIEW

File: source/components/Header/components.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const Container = styled.header`
+  height: 4rem;
```
Comment (leshnevskyi) [inline source/components/Header/components.tsx:6]: Liking using rems here. Good job mate!

File: source/components/Header/components.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  padding: 0 1.5rem;
+  position: relative;
+  z-index: 999;
```
Comment (leshnevskyi) [inline source/components/Header/components.tsx]: Makes sense to set the value to 1.

File: source/components/Header/components.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #2c2c2c;
```
Comment (leshnevskyi) [inline source/components/Header/components.tsx]: Would be nice to add the color to the palette.

File: source/components/Header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import {Container} from './components';
+
+const Header = () => {
+	return (
+		<>
```
Comment (leshnevskyi) [inline source/components/Header/index.tsx]: No need to use React Fragment here.

**Review by leshnevskyi — APPROVED** [review]
Good job mate! Just a few small tweaks would make it even better!

File: source/components/Header/components.tsx (hunk 2 above)
Comment (Danylo-tk) [reply source/components/Header/components.tsx]: All set!

File: source/components/Header/components.tsx (hunk 3 above)
Comment (Danylo-tk) [reply source/components/Header/components.tsx]: Added this color to the palette as ghostgrey

Comment (leshnevskyi) [reply source/components/Header/components.tsx]: Cool stuff, liking it 😁

File: source/components/Header/index.tsx (hunk 4 above)
Comment (Danylo-tk) [reply source/components/Header/index.tsx]: Already deleted it

**Pull request merged**